package/gluon-config-mode-geo-location-osm/Makefile

```diff
@@ -13,7 +13,7 @@ define Package/gluon-config-mode-geo-location-osm/install
 	$(Gluon/Build/Install)
 
 	$(INSTALL_DIR) $(1)/lib/gluon/config-mode/www/static/
-	$(LN) /lib/gluon/web/www/static/gluon-web-osm.js $(1)/lib/gluon/config-mode/www/static/
+	$(LN) /lib/gluon/web/www/static/gluon-web-osm.min.js $(1)/lib/gluon/config-mode/www/static/gluon-web-osm.js
```

Comment:
...and this are hardly right.
I do not see yet why this file lies in `/lib/gluon/web/www/static/`, yet.

The config-mode uses it from `/lib/gluon/config-mode/www/static`, so why put it here as well?